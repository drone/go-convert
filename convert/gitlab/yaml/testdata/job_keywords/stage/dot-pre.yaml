# https://docs.gitlab.com/ee/ci/yaml/#stage-pre

stages:
  - build
  - test

job1:
  stage: build
  script:
    - echo "This job runs in the build stage."

first-job:
  stage: .pre
  script:
    - echo "This job runs in the .pre stage, before all other stages."

job2:
  stage: test
  script:
    - echo "This job runs in the test stage."