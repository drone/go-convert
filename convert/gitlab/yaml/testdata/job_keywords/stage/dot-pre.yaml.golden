stages:
    - build
    - test
first-job:
    script:
        - echo "This job runs in the .pre stage, before all other stages."
    stage: .pre
job1:
    script:
        - echo "This job runs in the build stage."
    stage: build
job2:
    script:
        - echo "This job runs in the test stage."
    stage: test
